deployment/postgres-init-db/init-aerie.sh

```diff
@@ -29,7 +29,7 @@ psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-E
 
   \echo 'Initializing aerie_commanding database...'
   CREATE DATABASE aerie_commanding;
-  GRANT ALL PRIVILEGES ON DATABASE aerie_ui TO aerie;
+  GRANT ALL PRIVILEGES ON DATABASE aerie_commanding TO aerie;
```

Comment:
I am curious why this wasn't an issue before - do we not use the `aerie` user when our services interact with the database?

Reply:
I am not sure as we are using `aerie` user in the docker file for the command expansion container.

Reply:
Feel free to merge - we can dig into our database users and how they work offline.